.github/workflows/deployment.yml

```diff
@@ -78,12 +78,12 @@ jobs:
       - name: Timezone 🌐
         run: echo "Timezone set to ${{ env.TIMEZONE }}"
 
-  sql-migrations:
-    name: SQL Migrations 🗄️
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+  database:
```

Comment:
idm getting this in now for the prod deployment, but can we keep migrations in the name please as that's more descriptive